README.rst

```diff
@@ -98,8 +101,9 @@ Get historical data for specific stations, using Python:
         prefer_local=True
     )
 
+    # Collect and analyse data here.
     for df in observations.collect_data():
-        # analyse the station here
+        pass
```

Comment:
What should this loop do?

Reply:
It would collect the data, I think. Maybe printing some output of the collected data would fit here.

Reply:
Thanks. I will say
```
# Collect and analyse data here.
for df in observations.collect_data():
    print(df)
```
here.